emails/templates/ng_alert_notification.txt

```diff
@@ -1,38 +1,59 @@
-[[HiddenSubject .Subject "[[.Title]]"]]
-
-[[.Title]]
-----------------
-
-[[ .Alerts | len ]] alert[[ if gt (len .Alerts) 1 ]]s[[ end ]] for
-[[ range .GroupLabels.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ if gt (len .Alerts.Firing) 0 ]]([[ .Alerts.Firing | len ]]) Firing[[ end ]]
-[[ range .Alerts.Firing ]]
-Labels:
-[[ range .Labels.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ if gt (len .Annotations) 0 ]]
-Annotations:
-[[ end ]]
-[[ range .Annotations.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ end ]][[ if gt (len .Alerts.Resolved) 0 ]]([[ .Alerts.Resolved | len ]]) Resolved[[ end ]]
-[[ range .Alerts.Resolved ]]
-Labels:
-[[ range .Labels.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ if gt (len .Annotations) 0 ]]
-Annotations:
-[[ end ]]
-[[ range .Annotations.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ end ]]View your Alert rule:
-[[.RuleUrl]]
-
-Go to the Alerts page:
-[[.AlertPageUrl]]
+[[- define "__default_message" -]]
+You have [[ len .Alerts.Firing ]] firing alert(s), and [[ len .Alerts.Resolved ]] resolved alert(s) for [[ .GroupLabels ]]
+
+[[ if .Alerts.Firing -]]
+([[ len .Alerts.Firing ]]) FIRING
+-----------
+[[ template "__default_alerts_summarize" .Alerts.Firing ]]
+[[- end ]]
+
+[[ if .Alerts.Resolved -]]
+([[ len .Alerts.Resolved ]]) RESOLVED
+-------------
+[[ template "__default_alerts_summarize" .Alerts.Resolved ]]
+[[- end ]]
+Go to the Alerts page: [[ .AlertPageUrl ]]
+[[- end -]]
+
+[[- define "__default_alerts_summarize" -]]
+[[- range . ]]
+[[ template "__default_alert_labels" . ]]
+[[ template "__default_alert_annotations" . ]]
+[[- end ]]
+[[- end -]]
+
+[[- define "__default_alert_labels" -]]
+Labels: [[ .Labels.SortedPairs ]]
```

Comment:
In HTML email we put labels on a new line each. I think we should do the same here. WDYT?

Reply:
It's quite nice to have them on the same line when there are just a couple of labels, but what if we put them on multiple lines once we have more than 3 labels?

Reply:
```
You have 2 firing alert(s), and 0 resolved alert(s) for grafana_folder=Test Folder

(2) FIRING
-----------

Labels: alertname=Test 1, grafana_folder=Test Folder
Summary: This is a summary
Description: This is a description

Labels: 
    alertname = Test 2
    Label 1 = Value 1
    Label 2 = Value 2
    grafana_folder = Test Folder
Summary: This is a summary
Description: This is a description



Go to the Alerts page: http://127.0.0.1:3000/alerting/list?alertState=firing&amp;view=state

Sent by Grafana v10.1.0-pre (c) 2023 Grafana Labs
```